docs/visual-basic/programming-guide/language-features/procedures/optional-parameters.md

```diff
@@ -35,7 +35,7 @@ You can specify that a procedure parameter is optional and no argument has to be
  The following syntax shows a procedure declaration with an optional parameter:  
   
 ```  
```

Comment:
can you also add the language identifier here and on the next change?
from \`\`\` to \`\`\`vb

Reply:
@mairaw Done! If you have any more suggestions, let me know!